src/os/ui/state/stateexport.js

```diff
@@ -1,5 +1,6 @@
 goog.provide('os.ui.state.StateExportCtrl');
 goog.provide('os.ui.state.stateExportDirective');
+goog.provide('os.ui.state.unsupportedState');
```

Comment:
As we are progressing with the module transformation work, adding multiple provides to a file is just making someone have to ensure we're happy with it getting decomposed when we modularize the file in the future. I'd prefer adding this to a state utility class or another file.

Reply:
Agree. Any new code should avoid adding multiple provides to files.